tasks/doc.py

```diff
@@ -1,6 +1,7 @@
 from . import tool
 
 install_agent: str = f"Install the Agent (default {tool.get_default_agent_install()})."
+install_docker: str = "Install docker on the created VM, default to False since default AMI includes Docker"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I would not expose it here, as we don't allow to change the AMI here we should not allow to install docker